crates/rooch/src/commands/statedb/commands/genesis_ord.rs

```diff
@@ -472,13 +472,17 @@ fn produce_ord_updates(tx: SyncSender<BatchUpdatesOrd>, input: PathBuf, batch_si
     drop(tx);
 }
 
-fn gen_inscription_ids_update(index: u64, inscription_id: InscriptionID) -> (KeyState, State) {
-    let key = bcs::to_bytes(&index).expect("bcs to_bytes u64 must success.");
-    let key_state = KeyState::new(key, u64::type_tag());
-
-    let state = inscription_id.into_state();
-
-    (key_state, state)
+fn gen_inscription_ids_update(
+    index: u64,
+    inscription_id: InscriptionID,
+) -> (FieldKey, ObjectState) {
+    //let key = bcs::to_bytes(&index).expect("bcs to_bytes u64 must success.");
+    //TODO we need to get the TableVec object id from args
```

Comment:
Shouldn't it be Table, not TableVec, as the parent id?

Reply:
It is `inscriptions: TableVec<InscriptionID>` field of InscriptionStore

Reply:
> It is `inscriptions: TableVec<InscriptionID>` field of InscriptionStore

Inside TableVec is a Table, my original implementation:

1. put index:inscription in Table
2. using Table make TableVec

Reply:
yes. need to pass the id here